templates/my.cnf.erb

```diff
@@ -6,8 +6,10 @@
 <%     v.sort.map do |ki, vi| -%>
 <%       if ki == 'ssl-disable' or (ki =~ /^ssl/ and v['ssl-disable'] == true) -%>
 <%         next %>
-<%       elsif vi == true or vi == '' -%>
+<%       elsif [true, '', 'on'].include?(vi) -%>
 <%=        ki %>
+<%       elsif [false, 'off'].include?(vi) -%>
```

Comment:
I wonder if we would need `:false` here... sometimes puppet passes symbols instead of booleans.